Source/Orts.Simulation/Simulation/Physics/Train.cs

```diff
@@ -913,6 +916,10 @@ public Train(Simulator simulator, BinaryReader inf)
             InitialSpeed = inf.ReadSingle();
             IsPathless = inf.ReadBoolean();
 
+ /*           var hasEOT = inf.ReadInt32();
+            if (hasEOT == 1)
+                EOT = new EOT(inf, this);*/
```

Comment:
Did you forgot to uncomment this, or save/restore does not work yet?

Reply:
Save/restore works, but it does not use this. Lines can be removed.